common/src/main/java/org/figuramc/figura/FiguraMod.java

```diff
@@ -49,6 +48,13 @@ public class FiguraMod {
     public static boolean parseMessages = true;
     public static boolean processingKeybind;
 
+    public static boolean debugModeEnabled() {
+        if (Configs.DEBUG_MODE == null) return false;
+        Boolean b = Configs.DEBUG_MODE.value;
+        if (b == null) return false;
+        return b;
```

Comment:
you do know you could simplify this entire method to
```suggestion
   return b == true;
```
lol